webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'ui-react.js',
+    },
+    module: {
+        rules: [{
+            test: /\.scss$/,
+            use: [{
+                loader: 'style-loader',
+            }, {
+                loader: 'css-loader',
+            }, {
+                loader: 'sass-loader',
```

Comment:
postcss to be added in a follow-up, and more tweaks
We can take some inspiration of production settings from well documented CRA webpack config:
https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/config/webpack.config.prod.js